js/check_config.js

```diff
@@ -54,21 +50,18 @@ function checkConfigFile() {
 	Log.info(Utils.colors.info("Checking file... "), configFileName);
 
 	// I'm not sure if all ever is utf-8
-	fs.readFile(configFileName, "utf-8", function (err, data) {
-		if (err) {
-			throw err;
-		}
-		const messages = linter.verify(data);
-		if (messages.length === 0) {
-			Log.info(Utils.colors.pass("Your configuration file doesn't contain syntax errors :)"));
-		} else {
-			Log.error(Utils.colors.error("Your configuration file contains syntax errors :("));
-			// In case the there errors show messages and return
-			messages.forEach((error) => {
-				Log.error("Line", error.line, "col", error.column, error.message);
-			});
+	const configFile = fs.readFileSync(configFileName, "utf-8");
```

Comment:
In general, you are right, but since we throw the error afterward it is the same as not having a try catch

```
fs.readFile(configFileName, "utf-8", function (err, data) {
    if (err) {
        throw err;
    }
```